### PR TITLE
Garrett/pool

### PR DESCRIPTION
**Pull request by GarrettJMU**

### CODE REVIEW

File: src/components/AssetMenu/index.tsx [hunk 1]
```diff
@@ -31,66 +14,39 @@ const TokenSymbols = Tokens.reduceRight((prev: any, { symbol, logoURI }) => {
...
-          <MenuButton
-            _active={{ bg: '5c8abc' }}
-            _hover={{ bg: '#5c8abc' }}
-            _focus={{ bg: '#5c8abc' }}
+const AssetMenu = () => (
+  <Menu>
+    {({ isOpen }) => (
+      <>
+        <MenuButton
+          _active={{ bg: '5c8abc' }}
```
Comment (jmichaliga) [inline src/components/AssetMenu/index.tsx:22]: missing '#'?

File: src/pages/stake.tsx [hunk 2]
```diff
@@ -57,20 +56,23 @@ export default function Deposit() {
             <>
               <Button
                 _hover={{ color: 'black', background: 'white' }}
-                backgroundColor="#06927b"
-                color="white"
+                backgroundColor="#FFD500"
```
Comment (jmichaliga) [inline src/pages/stake.tsx:59]: `ukraineYellow` I think exists

**Review by jawauntb — APPROVED** [review]

**Review by jmichaliga — COMMENTED** [review]
looks promising! too sleepy to review fully

File: src/components/BoxDepositBox/index.tsx [hunk 3]
```diff
@@ -1,29 +1,93 @@
...
+  const handleStaking = async () => {
+    if (account) {
+      const signer = new ethers.providers.Web3Provider(
+        window.ethereum
+      ).getSigner()
+
+      if (prizePool) {
+        const user = new User(prizePool.prizePoolMetadata, signer, prizePool)
+
+        return user.approveDeposits().then(() => {
```
Comment (dylandesrosier) [inline src/components/BoxDepositBox/index.tsx:27]: `approveDeposits` could fail or return an error. Probably better to do something like:
```js
const approveTx = await user.approveDeposits()
const approveReceipt = await approveTx.wait()
const depositTx = await user.depositAndDelegate(...)
```
and check the status after waiting

File: src/components/BoxUnstakeBox/index.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  const tokenBalance = useTokenBalance(ticketTokenAddress, account)
+
+  const handleStaking = async () => {
+    if (account) {
+      const signer = new ethers.providers.Web3Provider(
+        window.ethereum
+      ).getSigner()
+
+      if (prizePool) {
+        const user = new User(prizePool.prizePoolMetadata, signer, prizePool)
```
Comment (dylandesrosier) [inline src/components/BoxUnstakeBox/index.tsx:29]: Should probably add a check to see if the user has at least the amount they're trying to withdraw

Comment (dylandesrosier) [reply src/components/BoxUnstakeBox/index.tsx:29]: An extra check to make sure the signer is on the right chain id would also be nice

**Pull request merged**